packages/odie-client/src/components/message/message-content.tsx

```diff
@@ -27,50 +29,72 @@ export const MessageContent = forwardRef<
 			isLastFeedbackMessage,
 			isLastMessage,
 			isLastUserMessage,
+			isNextMessageFromSameSender,
+			displayChatWithSupportLabel,
 		},
 		ref: ForwardedRef< HTMLDivElement >
 	) => {
+		const { shouldUseHelpCenterExperience } = useOdieAssistantContext();
 		const isUser = message.role === 'user';
+		const isWapuu = message.role === 'bot';
+		const isHuman = message.role === 'business';
+
 		const messageClasses = clsx(
 			'odie-chatbox-message',
-			isUser ? 'odie-chatbox-message-user' : 'odie-chatbox-message-wapuu',
+			isUser && 'odie-chatbox-message-user',
+			isWapuu && 'odie-chatbox-message-wapuu',
+
+			shouldUseHelpCenterExperience && isUser && 'odie-chatbox-message-user',
+			shouldUseHelpCenterExperience && isHuman && 'odie-chatbox-message-business',
+			shouldUseHelpCenterExperience && isWapuu && 'odie-chatbox-message-wapuu',
```

Comment:
These seem to be redundant besides the `isHuman` one.

Also, would it be easier if it were something like:

```
`odie-chatbox-message-${ message.role }`
```

This would be more easier to understand IMO. The only change you would have to make is update the CSS 'odie-chatbox-message-wapuu' to 'odie-chatbox-message-bot' to match

Reply:
Good suggestion, I have addressed, `isHuman` still mapped to `business`, all the others are dinamyc.